Tribler/Main/vwxGUI/list_body.py

```diff
@@ -763,6 +763,9 @@ def OnCollapse(self, item=None, raise_events=True, from_expand=False):
 
     @warnWxThread
     def OnChange(self, scrollToTop=False):
+        if GUIUtility.getInstance().utility.abcquitting:
```

Comment:
This is to fix the mysterious `PyDeadObject` problem in `ListBody`. See the first few test results for the backtrace.

This problem happens during the shutdown.  I tracked it down to this `SetupScrolling()`. This is a wx function which schedules `_SetupAfter()` with `wx.CallAfter()`, and this `_SetupAfter()` is the one that raised the `PyDeadObject` error.

The only place I found that calls `SetupScrolling()` is this `OnChannge()` in `ListBody`. Because this new RSS parser I made is too fast (compared to the old one) that in the rss_import test, you can see tens of torrents showing up simultaneously on the "Channel Manage" page, I suspect that this performance improvement might be the issue.

In short, the solution to fix this `PyDeadObject` problem is to not let `ListBody` update itself during the shutdown. So I added this line.